### PR TITLE
improve logging

### PR DESCRIPTION
**Pull request by vadim-moz**

### CODE REVIEW

File: spec/unit/worker_spec.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ class JobClass; end
         job.should respond_to(:fail).with(2).arguments
         job.should_receive(:fail) do |group, message|
           last_line = message.split("\n").last
-          expect(last_line).to match(/base\.rb:\d+:in `around_perform'/)
+          expect(last_line).to match(/base\.rb:\d+:in `perform'/)
```
Comment (vadim-moz) [inline spec/unit/worker_spec.rb]: This test is a bit fragile, since it depends on whether call `around_perform` is the first thing in `perform` or not.

File: lib/qless/worker/base.rb [hunk 2]
```diff
@@ -166,9 +170,9 @@ def try_complete(job)
         #   - The job is being worked on by another worker
         #   - The job has been cancelled
         #
-        # We don't want to (or are able to) fail the job with this error in
+        # We don't want to (or are able to) fail the job with this warning in
```
Comment (myronmarston) [inline lib/qless/worker/base.rb]: The "error" in this sentence was a reference to the Exception being rescued above, not to the log level used below.  As such, I think it's less confusing to revert the change in this line.

File: spec/unit/worker_spec.rb (hunk 1 above)
Comment (myronmarston) [inline spec/unit/worker_spec.rb]: WDYT about extracting the regex into a variable named `first_line_in_perform_method_regex` or similar?  Then it's explicitly labeled.

**Comment by myronmarston** [comment]
LGTM

File: lib/qless/worker/base.rb (hunk 2 above)
Comment (vadim-moz) [inline lib/qless/worker/base.rb]: done

File: spec/unit/worker_spec.rb (hunk 1 above)
Comment (vadim-moz) [inline spec/unit/worker_spec.rb]: done

**Comment by myronmarston** [comment]
LGTM

**Comment by vadim-moz** [comment]
Thank you!
